### PR TITLE
Allows dynamic elements to be added at a specific position.

### PR DESCRIPTION
**Pull request by mikesol**

### CODE REVIEW

**Comment by mikesol** [comment]
smöl example [here](chilly-doctor.surge.sh) and in `examples/insert`

**Review by PureFunctor — APPROVED** [review]

File: src/Deku/Interpret.js [hunk 1]
```diff
@@ -9,11 +9,12 @@ export const unSetHydrating = (state) => () => {
...
 export const attributeParent_ = (a) => (state) => () => {
 	// only attribute if it is not attributed already
+	console.log(a);
 	if (!state.units[a.id].main.parentNode) {
 		if (
-			a.pos.value0 instanceof number &&
+			typeof a.pos.value0 === 'number' &&
 			state.units[a.parent].main.children[a.pos.value0]
-		) {
+		) {console.log("ibefore", a.pos.value0);
```
Comment (PureFunctor) [inline src/Deku/Interpret.js:17]: Small nit, are these `console.log`s intentional?

**Review by PureFunctor — CHANGES_REQUESTED** [review]

File: src/Deku/Interpret.js (hunk 1 above)
Comment (mikesol) [reply src/Deku/Interpret.js:17]: gloups, nope, removed!

**Review by PureFunctor — APPROVED** [review]
:shipit:

**Pull request merged**